optimize/util/optimize-commons/src/test/java/io/camunda/optimize/CodeQualityTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize;
+
+import io.camunda.optimize.service.exceptions.OptimizeRuntimeException;
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class CodeQualityTest {
+  @Test
+  public void testNoForbiddenAnnotationUsed() {
```

Comment:
I think the naming here is totally fine. Not super important, but another thought here is that we could instead test against the imports too? So rather than just forbidden annotations, could we also include `import lombok.` in the list of forbidden strings?